Tests/MySQLTests/MySQLTests.swift

```diff
@@ -249,4 +250,20 @@ class MySQLTests: XCTestCase {
             XCTFail("Testing transaction failed: \(error)")
         }
     }
+
+    func testBlob() throws {
+        let c = try mysql.makeConnection()
+        try c.execute("DROP TABLE IF EXISTS blobs")
+        try c.execute("CREATE TABLE blobs (raw BLOB)")
+        let inputBytes = Node.bytes("Hello, World!".makeBytes()) // just need generic bytes of some type
```

Comment:
I think this test is too comfortable, the string would probably turn out fine even with UTF-8 decoding/encoding in place.

Try this instead (a bunch of invalid UTF-8 sequences):

```swift
let inputBytes = Node.bytes([0xc3, 0x28, 0xa0, 0xa1, 0xe2, 0x28, 0xa1, 0xe2, 0x82, 0x28, 0xf0, 0x28, 0x8c, 0xbc])
```